src/Aspire.ProjectTemplates/templates/aspire-empty/8.2/AspireApplication.1.ServiceDefaults/AspireApplication.1.ServiceDefaults.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
+    <PackageReference Include="AspNetCore.HealthChecks.ApplicationStatus" Version="ß!!REPLACE_WITH_LATEST_VERSION!!" />
```

Comment:
You don't want to use the replacement token here. Just put in the version number. This is causing the tests to fail currently.